pkgs/ok_http/example/lib/main.dart

```diff
@@ -147,3 +186,112 @@ class _BookListState extends State<BookList> {
         ),
       );
 }
+
+/// Example to demonstrate WebSocket communication with [OkHttpWebSocket],
+/// using the echo server at `wss://echo.websocket.org`.
+class WebSocketEcho extends StatefulWidget {
+  const WebSocketEcho({super.key});
+
+  @override
+  State<WebSocketEcho> createState() => _WebSocketEchoState();
+}
+
+class _WebSocketEchoState extends State<WebSocketEcho> {
+  // Contains a list of pairs [String message, bool isSent].
+  final List<List> _messages = [];
```

Comment:
Nit: can this be a map or some data class you make for clarity?

Reply:
Not proceeding as the WebSocket example has been removed.